asst1/asst1.py

```diff
@@ -0,0 +1,104 @@
+from collections import Counter
+
+def swapchars(string_thing):
+    # initialize a counter and count each letter's occurrence
+    count = Counter()
```

Comment:
You can acutally just call Counter() directly on the string! See the docs for details but you don't actually need the for loop underneath